docs/zh-CN/docs/data-operate/import/import-scenes/load-data-convert.md

```diff
@@ -195,7 +195,7 @@ under the License.
 | 1    | 100  | beijing   | 1.1  |
 | 2    | 200  | shanghai  | 1.2  |
 | 3    | 300  | guangzhou | 1.3  |
-| 4    | 400  | chongqing | 1.4  |
+| 4    | \N   | chongqing | 1.4  |
```

Comment:
In the result below, column k3 has 40,000, which is obviously the second column with the original data multiplied by 100. Why did you change it to null?
You should change the 4 in the first column to null

Reply:
Right,  I change it wrong